src/connectRoutes.js

```diff
@@ -115,8 +115,8 @@ export default (
   }
 
   const {
-    location: locationKey = 'location',
-    title: titleKey = 'title',
+    selectLocationState = state => state.location,
```

Comment:
OK, so the first of the two options then?

I think flow might complain about the overloading of `string | Function`. We'll see